app/main/views.py

```diff
@@ -128,12 +128,14 @@ def update(service_id, section):
                 update[question_id] = form.clean_data[question_id]
 
     if update:
-        service_loader.post(
+        api_response = service_loader.post(
             service['id'],
             update,
             session['username'],
             'admin app'
         )
+        if api_response.status_code is not 200:
+            return "Error: " + str(api_response.content)
```

Comment:
API response should be a JSON with "error" top-level key, so it might be worth trying to `api_response.json()['error']` and returning content only if that fails